ethereum/core/src/main/java/org/hyperledger/besu/ethereum/vm/Memory.java

```diff
@@ -98,9 +72,9 @@ private void checkByteIndex(final long v) {
     if (v < 0 || v >= MAX_BYTES) throw overflow(v);
```

Comment:
Is `MAX_BYTES` different now that we are using a straight byte[] instead of ArrayList<Bytes32>?

Reply:
Yes, it's max int now.  fixing